fastapi_pagination/ext/cassandra.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict, Mapping, Optional, Type, TypeVar
+
+from cassandra.cluster import SimpleStatement
+from cassandra.cqlengine import connection
+from cassandra.cqlengine.models import Model
+
+from ..api import create_page
+from ..bases import AbstractPage
+from ..utils import TParams, verify_params
+
+T = TypeVar("T", bound=Mapping[str, Any])
+
+
+def paginate_with_cursor(
+    model: Type[Model],
+    query_filter: Optional[Dict[Any, Any]] = None,
+    params: Optional[TParams] = None,
+    **kwargs: Any,
+) -> AbstractPage[Any]:
+    params, raw_params = verify_params(params, "cursor")
+
+    query_filter = query_filter or {}
+    stmt = SimpleStatement(
+        str(model.filter(**query_filter)),
+        fetch_size=raw_params.size,
+    )
+    conn = connection.get_connection()
+
+    cursor = conn.session.execute(
+        stmt, parameters={str(i): v for i, v in enumerate(query_filter.values())}, paging_state=raw_params.cursor
+    )
+    items = cursor.current_rows
+
+    return create_page(items, total=0, params=params, next_=cursor.paging_state)
```

Comment:
`total` can be removed, it's not required for `CursorPage`

Reply:
removed